docs/tutorial.md

```diff
@@ -0,0 +1,120 @@
+# urdf2webots Tutorial
+
+## Prerequisites:
+Make sure you have the newest version of urdf2webots installed. If you have an older version installed via pip, use this command:
+
+```
+pip install --no-cache-dir --upgrade urdf2webots
+```
+
+Alternatively, if you want to install from source, follow the instructions in the [Main README](../README.md)
+<br /> 
+<br /> 
+## Creating the URDF from git repo (optional if urdf already available)
+
+Example using this KUKA ros repo:
+https://github.com/ros-industrial/kuka_experimental
+
+```
+cd catkin_ws/src
+git clone https://github.com/ros-industrial/kuka_experimental
+cd ..
+catkin_make  # or catkin build  depending on what you use
+source devel/setup.bash
+```
+**Note:** If you want to convert this exact same robot, you will have to replace the file at
+kuka_experimental/kuka_lbr_iiwa_support/meshes/lbr_iiwa_14_r820/visual/base_link.dae
+With this:
+[base_link.dae](https://drive.google.com/file/d/1J0dVuDOW7k3wa6Gj0vpjKzlNMzQHOAfD/view?usp=sharing)
+<br /> 
+
+Navigate to the launch folder and open the launch file that launches your robot or displays it in Rviz. For this tutorial we chose the KUKA lbr iiwa robot. The launch file is:
+`/kuka_experimental/kuka_lbr_iiwa_support/launch/load_lbr_iiwa_14_r820.launch`
+Opening the launch file we look for the line, uploading the robot_description parameter, here it is this line:
+
+`<param name="robot_description" command="$(find xacro)/xacro.py '$(find kuka_lbr_iiwa_support)/urdf/lbr_iiwa_14_r820.xacro'" />`
+
+This tells you which xacro file and with what parameters we need to generate our urdf from.
+Next in your Terminal, navigate to the urdf folder specified in the launch file and enter following command:
+
+```
+cd src/kuka_experimental/kuka_lbr_iiwa_support/urdf
+rosrun xacro xacro --inorder -o model.urdf lbr_iiwa_14_r820.xacro
+```
+This will compile the urdf from the xacro file and save it as model.urdf in the same directory. If your launch file added parameters to the xacro calls, you need to add them here too.
```

Comment:
```suggestion
This will compile the URDF from the XACRO file and save it as model.urdf in the same directory. If your launch file added parameters to the XACRO calls, you need to add them here too.
```